osf/models/analytics.py

```diff
@@ -103,7 +123,7 @@ def update_counter(cls, page, node_info):
                     model_instance.date[date_string]['unique'] += 1
                 else:
                     # set the number of unique visitors to 1
-                    model_instance.date[date_string] = dict(unique=1)
+                    model_instance.date[date_string] = dict(unique=0)
```

Comment:
There is no way to to test this currently because necessary waterbutler callback code with the user discerning logic isn't on prod. So this doesn't really do anything.